### ISSUE TITLE
Unable to save image

### ISSUE THREAD

**Issue by johanbook** [rpearce/react-medium-image-zoom#259]
This issue is a: 

- Bug report

## Expected behavior
To be able to right click and choose `Save image as` for an image wrapped in `Zoom`

## Actual behavior
I could only save the page html

## Steps to reproduce behavior
Use
```jsx
<Zoom>
  <img src={source} />
</Zoom>
```

**Comment by rpearce** [comment]
@johanbook Thank you for reporting this. In the current version, there is a button covering the image. I will make sure this is not the case when I can circle back to the next version. https://github.com/rpearce/image-zoom/pull/244 got put on hold because of some unavoidable issues that needed to be rethought, and we've just had a baby, so I'm not doing any coding at the moment.

**Comment by johanbook** [comment]
Congratz to the baby @rpearce and thank you for this repo. If you point me in the right direction I make an attempt in creating a PR

**Comment by rpearce** [comment]
@johanbook It's unfotunately a bit more complicated than a simple PR, because the entire mechanism for "zooming" needs to change. THANK YOU for your offer of a PR, though.

**Comment by rpearce** [comment]
Similarly reported issue in https://github.com/rpearce/image-zoom/issues/271. I'll track it here.

**Comment by rpearce** [comment]
Update on v5 FYI: https://github.com/rpearce/image-zoom/discussions/274

**Comment by rpearce** [comment]
FYI: I'm working on this right now and not as part of that rewrite.

**Comment by rpearce** [comment]
There was no way to do this in the current version without breaking things. Here's the latest update: https://github.com/rpearce/image-zoom/discussions/274#discussioncomment-1616865

**Comment by rpearce** [comment]
Here's the latest update about the work being done that will resolve this: https://github.com/rpearce/react-medium-image-zoom/discussions/274#discussioncomment-2946565

**Comment by rpearce** [comment]
If anyone finds some spare time and wants to give the latest build a whirl to try this out:

```sh
npm i --save react-medium-image-zoom@5.0.0-beta2.6
```

And, if you aren't already,

```jsx
import Zoom from 'react-medium-image-zoom'
import 'react-medium-image-zoom/dist/styles.css'

// ... your other code

<Zoom>
  <MyImage />
</Zoom>
```

The updated [`v5-dev` README is here](https://github.com/rpearce/react-medium-image-zoom/tree/v5-dev#react-medium-image-zoom), and there are some breaking changes to the extra options passed to be aware of, if you're using the options.

Thanks for your time!

**Comment by rpearce** [comment]
Should be closed now with the release of `5.0.0`
